source/application/DbgHelp/DbgHelpTypes.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016- Hourglass Resurrection Team
+ * Hourglass Resurrection is licensed under GPL v2.
+ * Refer to the file COPYING.txt in the project root.
+ */
+
+#pragma once
+
+enum class DbgHelpArgType
+{
+    Unknown,
+    Char,
+    WideChar,
+    Char16,
+    Char32,
+    Short,
+    UnsignedShort,
+    Int,
+    UnsignedInt,
+    Long,
+    UnsignedLong,

```

Comment:
I think it's better to replace this stuff with variants meaning `int8_t`, `uint8_t`, `int16_t`, `uint16_t`, and so on.